frontend/packages/helm-plugin/integration-tests/features/helm/install-helm-chart.feature

```diff
@@ -0,0 +1,84 @@
+@helm
+Feature: Install the Helm Release
+              As a user, I want to install the helm release
```

Comment:
Indentation

Reply:
Updated automatically on save and not able to see any gherkin-lint issue